ai/ai-starter-kit/helm-chart/ai-starter-kit/files/multi-agent-ramalama.ipynb

```diff
@@ -0,0 +1,466 @@
+    "    stderr=subprocess.DEVNULL\n",
+    ")\n",
+    "\n",
+    "time.sleep(3)\n",
+    "\n",
+    "API_URL = \"http://localhost:8000\"\n",
+    "try:\n",
+    "    r = requests.get(f\"{API_URL}/v1/healthz\", timeout=5)\n",
+    "    print(\"API Status:\", r.json())\n",
+    "    print(f\"\\nOpenAI-compatible API running at: {API_URL}/v1\")\n",
+    "    print(f\"Health: {API_URL}/v1/healthz\")\n",
+    "    print(f\"Chat:   {API_URL}/v1/chat/completions\")\n",
+    "except Exception as e:\n",
+    "    print(f\"Warning: API wrapper not responding: {e}\")\n",
+    "    print(\"You may need to run the wrapper manually\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a411c015-c802-4ca1-81bb-3f4790d9626a",
+   "metadata": {},
+   "source": [
+    "### Cell 4 - Basic client + latency test\n",
+    "\n",
+    "Tests the OpenAI-compatible API with a simple chat request and measures latency."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3be634e2-a82f-42c9-8e31-57e6868a86ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os, time, requests, json\n",
+    "\n",
+    "USE_WRAPPER = True\n",
+    "BASE_URL = \"http://localhost:8000/v1\" if USE_WRAPPER else os.getenv(\"RAMALAMA_HOST\", \"http://127.0.0.1:8080\")\n",
+    "\n",
+    "def health():\n",
+    "    if USE_WRAPPER:\n",
+    "        r = requests.get(f\"{BASE_URL}/healthz\", timeout=10)\n",
+    "        print(\"Health:\", r.status_code, r.json())\n",
+    "    else:\n",
+    "        r = requests.get(f\"{BASE_URL}/v1/models\", timeout=10)\n",
+    "        print(\"Health:\", r.status_code, \"Models available:\", r.json().get('data', []))\n",
+    "\n",
+    "def chat(prompt, temperature=0.4, max_tokens=220):\n",
+    "    body = {\n",
+    "        \"model\": \"qwen2.5:1.5b\",\n",
+    "        \"temperature\": temperature,\n",
+    "        \"max_tokens\": max_tokens,\n",
+    "        \"messages\": [\n",
+    "            {\"role\": \"system\", \"content\": \"You are a helpful assistant. Be concise.\"},\n",
+    "            {\"role\": \"user\", \"content\": prompt},\n",
+    "        ],\n",
+    "        \"stream\": False\n",
+    "    }\n",
+    "    endpoint = f\"{BASE_URL}/chat/completions\"\n",
+    "    \n",
+    "    t0 = time.time()\n",
+    "    r = requests.post(endpoint, json=body, timeout=120)\n",
+    "    dt = time.time() - t0\n",
+    "    r.raise_for_status()\n",
+    "    \n",
+    "    response = r.json()\n",
+    "    content = response[\"choices\"][0][\"message\"][\"content\"]\n",
+    "    usage = response.get(\"usage\", {\"total_tokens\": \"estimated: \" + str(len(content.split()) + len(prompt.split()))})\n",
+    "    \n",
+    "    print(f\"\\nLatency: {dt:.2f}s  | usage: {usage}\")\n",
+    "    print(\"\\n---\\n\", content)\n",
+    "    return content\n",
+    "\n",
+    "health()\n",
+    "_ = chat(\"Say 'test ok' then give me one short fun fact about llamas.\")"
```

Comment:
I didn't get this expected output after running the cell. I think it's because the required model was not pulled.